src/metadata/metadata.service.ts

```diff
@@ -66,4 +67,19 @@ export class MetadataService {
       webhookUrl,
     });
   }
+
+  public async clearValue(
+    params: MetadataParamsDto,
+    opts: TransactionBaseDto
+  ): ServiceReturn<void> {
+    const { clear } = await this.findOne(params);
```

Comment:
With my other PR this should be:

```ts
  body: TransactionBaseDto
  ): ServiceReturn<void> {
    const { options } = extractTxOptions(body)
```

The conversion makes the DTO backwards compatible. It will be a type error so OK to merge if you merge before my other PR